server/src/main/java/org/opensearch/index/remote/RemoteSegmentTransferTracker.java

```diff
@@ -294,6 +294,11 @@ public void updateLatestLocalFileNameLengthMap(
         Collection<String> segmentFiles,
         CheckedFunction<String, Long, IOException> fileSizeFunction
     ) {
+        logger.debug(
```

Comment:
In OpenSearch codebase, in bunch of places, we have a check before logging with debug or trace level.

```
if (logger.isDebugEnabled()) {
    logger.debug("Some text");
}

```

I am not completely sure how it helps but better to check further.

Reply:
log4j doc mentions that this is not required anymore: https://github.com/apache/logging-log4j2/blob/2.x/log4j-api/src/main/java/org/apache/logging/log4j/Logger.java#L56

Reply:
Thanks for checking @sachinpkale.